src/cli.rs

```diff
@@ -299,6 +299,16 @@ pub enum EdgeAppCommands {
         #[arg(short, long)]
         path: Option<String>,
 
+        /// Edge app id. If not specified CLI will use the id from the manifest.
+        #[arg(short, long)]
+        app_id: Option<String>,
+    },
+    /// Deletes an edge app. This cannot be undone.
```

Comment:
Please capitalize. Edge App. Everywhere. If you see it incorrectly nearby please ifx it as well